app/src/main/java/net/squanchy/SquanchyApplication.java

```diff
@@ -30,6 +30,7 @@ public void onCreate() {
         setupTracking();
         FirebaseDatabase.getInstance().setPersistenceEnabled(true);
         TypefaceManager.init();
+        applicationComponent().service();
```

Comment:
Hm so is initializing the service producing side effects? That's a bit hard to grasp reading the code.

If that's the case, can we make sure that's explicit? For example by having a `void` method that needs to be invoked to start the beacon scans